app/models/concerns/pick_up_order.rb

```diff
@@ -0,0 +1,89 @@
+class PickUpOrder < ActiveRecord::Base
+  # rubocop:disable Metrics/BlockLength
```

Comment:
I don't get why we keep adding this cop to projects if we're going to disable it. IMHO it's not a useful cop anyway. Just my 2c.